taar/recommenders/locale_recommender.py

```diff
@@ -41,6 +44,6 @@ def can_recommend(self, client_data):
 
         return True
 
-    def recommend(self, client_data, limit):
-        client_locale = client_data.get('locale')
+    def recommend(self, client_data, limit, extra_data={}):
+        client_locale = client_data.get('locale') or extra_data.get('locale', None)
```

Comment:
Please comment that we have chosen to give priority to the client data from telemetry/hbase over the data source coming from AMO

Reply:
Done